sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1100,120 +1053,95 @@ event from a graph submission that was created without this property.
 
 ==== Graph Member Functions
 
-Table {counter: tableNumber}. Constructor of the `command_graph` class.
-[cols="2a,a"]
-|===
-|Constructor|Description
+===== Constructor of the `command_graph` class.
 
-|
 [source,c++]
 ----
 command_graph(const context& syclContext,
               const device& syclDevice,
               const property_list& propList = {});
 ----
-|Creates a SYCL `command_graph` object in the modifiable state for context
+
+_Effects:_ Creates a SYCL `command_graph` object in the modifiable state for context
 `syclContext` and device `syclDevice`. Zero or more properties can be provided
 to the constructed SYCL `command_graph` via an instance of `property_list`.
 
-Constraints:
+_Constraints:_
 
 * This constructor is only available when the `command_graph` state is
   `graph_state::modifiable`.
 
-Parameters:
+* `syclContext` is an immutable characteristic of the graph.
 
-* `syclContext` - Context which will be associated with this graph and all
-  nodes within it. This is an immutable characteristic of the graph.
+* `syclDevice` must be associated with `syclContext`.
 
-* `syclDevice` - Device that all nodes added to the graph will target,
-  an immutable characteristic of the graph. Must be associated with
-  `syclContext`.
-
-* `propList` - Optional parameter for passing properties. Valid `command_graph`
+* Valid `command_graph`
   constructor properties are listed in Section <<graph-properties, Graph Properties>>.
 
-Exceptions:
+_Throws:_
 
-* Throws synchronously with error code `invalid` if `syclDevice` is not
+* Synchronously with error code `invalid` if `syclDevice` is not
 associated with `syclContext`.
 
-* Throws synchronously with error code `invalid` if `syclDevice`
+* Synchronously with error code `invalid` if `syclDevice`
   <<device-info-query, reports this extension as unsupported>>.
 
-|
 [source,c++]
 ----
 command_graph(const queue& syclQueue,
               const property_list& propList = {});
 ----
-|Simplified constructor form where `syclQueue` provides the device and context.
+
+_Effects:_ Simplified constructor form where `syclQueue` provides the device and context.
 Zero or more properties can be provided to the constructed SYCL `command_graph`
 via an instance of `property_list`.
 
-Constraints:
+_Constraints:_
 
 * This constructor is only available when the `command_graph` state is
   `graph_state::modifiable`.
 
-Parameters:
-
-* `syclQueue` - Queue which provides the SYCL device and context for the graph,
+* Queue only provides the SYCL device and context for the graph,
```

Comment:
I think this is an effect rather than a constraint, as it's more specifying the behaviour of the constructor.